GraspObject obj
#sampling method for experiments
bool sample
bool non_random
---
geometry_msgs/PoseStamped object_pose
geometry_msgs/PoseStamped[] palm_goal_pose_world
sensor_msgs/JointState[] allegro_joint_state
bool[] is_top_grasp